src/UnicodeNormDApp/Program.cs

```diff
@@ -11,10 +11,10 @@ class Program
     {
         static void Main(string[] args)
         {
-
-            var httpClient = new WebClient();
-
-            var data = httpClient.DownloadString("http://www.unicode.org/Public/UCD/latest/ucd/NormalizationTest.txt");
+            using var httpClient = new HttpClient();
+            using var response = httpClient.GetAsync("http://www.unicode.org/Public/UCD/latest/ucd/NormalizationTest.txt").Result;
+            using var content = response.Content;
+            var data = content.ReadAsStringAsync().Result;
```

Comment:
```suggestion
            var data = await new HttpClient().GetStringAsync("http://www.unicode.org/Public/UCD/latest/ucd/NormalizationTest.txt");
```
We can just make `Main` `async` here.